tests/build.gradle

```diff
@@ -22,8 +22,6 @@ configure(subprojects - project(":tests:gdx-tests-android")) {
     sourceSets.main.java.srcDirs = ["src"]
     sourceSets.main.resources.srcDirs = ["res"]
 
-    compileJava.options.encoding = 'UTF-8'
```

Comment:
Again, why is the encoding being reverted to platform-specific here? Is it set in the root maybe? I use
```gradle
[compileJava, compileTestJava, javadoc]*.options*.encoding = 'UTF-8'
```
in all of my projects, personally.